grassroots-backend/src/auth/Auth.controller.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable grassroots/controller-routes-return-dtos */
```

Comment:
Why was this needed?

Reply:
Ah, the eslint rule thinks loadEnvironmentVariables is a controller route. This is arguably broken, but I think ensuring that controller methods are all routes is probably actually reasonable.

In any case, see my comment below on loadEnvironmentVariables - I don't think we need it, which means we don't need this line.